jekyll/assets/js/instantSearch.js

```diff
@@ -6,4 +6,66 @@ var search = instantsearch({
   routing: true
 });
 
+// initialize SearchBox
+search.addWidget(
+  instantsearch.widgets.searchBox({
```

Comment:
I think we should consider adding the `queryHook` option to here and [here](https://github.com/circleci/circleci-docs/pull/2720/files#diff-50bd272b9e398cba0799cac3f16c7b3dR22) so we can throttle the results. It looks like a request is being made to algolia on every keypress.

Reply:
Actually, for the instant search results loading with each keystroke, I think the current behavior is preferable. If it becomes a performance issue, maybe we can revisit this?

Reply:
I'm worried that we are going to run into this if we have a few people using this at the same time: https://community.algolia.com/places/pricing.html#rate-limits

It's also considered best practices to throttle requests like this rather than making them on every keystroke.